no-browser-site-scrapers/Curative/index.js

```diff
@@ -16,12 +16,20 @@ module.exports = async function GetAvailableAppointments() {
                     body += chunk;
                 });
                 res.on("end", () => {
-                    response = JSON.parse(body);
+                    try {
+                        response = JSON.parse(body);
+                    } catch (e) {
+                        console.error("JSON.parse failed: " + e);
+                    }
                     resolve(response);
                 });
             });
         });
-        rawData[id] = await p;
+        try {
```

Comment:
does this part do anything? I'm a little confused

Reply:
I removed this